src/styles.scss

```diff
@@ -6,15 +6,50 @@
 :root {
   --mdc-filled-text-field-container-color: rgba(255, 255, 255, 0.08);
   --mdc-filled-text-field-input-text-color: rgba(255, 255, 255, 0.87);
-  --mdc-filled-text-field-input-text-placeholder-color: rgba(
-    255,
-    255,
-    255,
-    0.87
-  );
-  --mdc-filled-text-field-label-text-color: var(--Gray-500, #6b6b83);
-
-  --mdc-filled-text-field-button-disabled-color: rgba(5, 5, 206, 0.356);
+  --mdc-filled-text-field-input-text-placeholder-color: rgba(255, 255, 255, 0.87);
+  --mdc-filled-text-field-active-indicator-color: #6002ee;
+  --mdc-filled-text-field-label-text-color: var(--Gray-500, #6B6B83);
+}
+
+.custom__modal__forgotpassword {
```

Comment:
Poderia deixar o nome da classe como ".custom__modal" apenas, aí daria para reutilizar no modal de how to play.